workflow/executor/agent.go

```diff
@@ -368,6 +368,9 @@ func (ae *AgentExecutor) executePluginTemplate(ctx context.Context, tmpl wfv1.Te
 			return 0, err
 		} else if reply.Node != nil {
 			*result = *reply.Node
+			if reply.Node.Phase == wfv1.NodeSucceeded {
```

Comment:
Should this include failed and errored nodes?

Reply:
I don't think so. Because in failure, the user needs to `explicitly give` the delay of the requeue.